packages/runtime/src/runtimes/react/hooks/use-resolved-props.ts

```diff
@@ -16,6 +16,27 @@ import { useStylesheetFactory } from './use-stylesheet-factory'
 
 import { resolvableRecord } from '../resolvable-record'
 
+function propErrorHandlingProxy(
+  resolvable: Resolvable<unknown>,
+  def: ControlDefinition,
+  propName: string,
+): Resolvable<unknown> {
+  return {
+    ...resolvable,
+    readStable: () => {
+      try {
+        return resolvable.readStable()
+      } catch (err) {
+        const defaultValue = (def.config as any)?.defaultValue
+        console.error(
+          `Error reading value for prop "${propName}", using \`${defaultValue}\` instead. Error: ${err}`,
```

Comment:
Sorry I didn't pay attention to this initially, I'd log the error as an object here and also log the control type/definition:

```typescript
console.error(
    `Error reading value for prop "${propName}", falling back to \`${defaultValue}\`.`,
    { control: def, error: err }
)
```